cloudpickle/cloudpickle.py

```diff
@@ -1311,14 +1320,33 @@ def dump(self, obj):
     def __init__(self, file, protocol=None, buffer_callback=None):
         if protocol is None:
             protocol = DEFAULT_PROTOCOL
-        super().__init__(file, protocol=protocol, buffer_callback=buffer_callback)
         # map functions __globals__ attribute ids, to ensure that functions
         # sharing the same global namespace at pickling time also share
         # their global namespace at unpickling time.
         self.globals_ref = {}
         self.proto = int(protocol)
+        self.super_pickler().__init__(
+            file, protocol=protocol, buffer_callback=buffer_callback
+        )
+
+    def super_pickler(self):
```

Comment:
```suggestion
    def _super_pickler(self):
```

(nit: to mark it's for internal usage)